### PR TITLE
Fixed lab 2

### PR DESCRIPTION
**Pull request by WinApiMan**

### CODE REVIEW

File: Lab2/src/BusinessLogic/ReadWriteServices/DataBaseFromFile.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿using AutoMapper;
+using BusinessLogic.Interfaces;
+using BusinessLogic.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using Taxi.DAL.Interfaces;
+using Taxi.DAL.Models;
+
+namespace BusinessLogic.Services
```
Comment (alexey62soft) [inline Lab2/src/BusinessLogic/ReadWriteServices/DataBaseFromFile.cs:10]: Please, rename folder too)

File: Lab2/src/Lab2Console/TaxiServices/OrderConsoleService.cs [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+using System.Data.Common;
+using System.Linq;
+using System.Threading.Tasks;
+using Taxi.BusinessLogic.Processings;
+using Taxi.BusinessLogic.Validations;
+using Taxi.ConsoleUI.Enums;
+using Taxi.ConsoleUI.Interfaces;
+using ILogger = NLog.ILogger;
+
+namespace Taxi.ConsoleUI.TaxiServices
```
Comment (alexey62soft) [inline Lab2/src/Lab2Console/TaxiServices/OrderConsoleService.cs:14]: Just Service (and folder too)

File: Lab2/src/BusinessLogic/ReadWriteServices/DataBaseFromFile.cs (hunk 1 above)
Comment (WinApiMan) [reply Lab2/src/BusinessLogic/ReadWriteServices/DataBaseFromFile.cs:10]: change namespace to the name of the folder or name of the folder to namespace?

Comment (WinApiMan) [reply Lab2/src/BusinessLogic/ReadWriteServices/DataBaseFromFile.cs:10]: I already have a Service folder

**Comment by alexey62soft** [comment]
Please, upload the report too.

**Comment by alexey62soft** [comment]
I see, merge it into master.

**Pull request merged**